### ISSUE TITLE
Reveal different questions based on selected option in a dropdown in a form

### ISSUE THREAD

**Issue by wes-otf** [HyphaApp/hypha#3852]
Something that is frequently requested by staff is the option to have something similar to the `Group Fields` block but instead of being show/hide, different questions are revealed based off of what is selected in the dropdown.

**Comment by frjo** [comment]
Yes, it has come up a few times over the last two years or so. Would be a very good feature.

There are some work to implement this but the hardest part i think will be to design a nice UI for it. Making it simple to use for the staff creating forms.
